lib/util.mjs

```diff
@@ -56,3 +57,21 @@ export function replace(values, valueProp, replacerMap) {
     }
   });
 }
+
+export async function execaNodeAndLog(cmd, args, cwd) {
```

Comment:
Could we do something like this?

```
export function withLog(fn) {
  return (cmd, args, cwd) => {
    const opts = {cwd};

    opts.stdout = process.stdout;
    opts.stderr = process.stderr;

    return fn(cmd, args, opts);
  }
}
```

Then call it with something like:

```
withLog(execa)(cmd, args, cwd);
withLog(execaNode)(cmd, args, cwd);
```

Fine if you prefer it like this for simplicity. There may also be a better way to generalize.

Reply:
Good point! I added an optional argument to specify whether the command is meant to be run on Node or not, which allows us to use just one function.